lupa/_lupa.pyx

```diff
@@ -1399,12 +1402,13 @@ cdef bint call_python(LuaRuntime runtime, lua_State *L, py_object* py_obj) excep
 
 cdef int py_call_with_gil(lua_State* L, py_object *py_obj) with gil:
     cdef LuaRuntime runtime = None
+    runtime = <LuaRuntime?>py_obj.runtime
```

Comment:
Do we know why? I mean, we have to have some tests for that then because now the tests went just fine even without catching?